configure.ac

```diff
@@ -64,12 +64,18 @@ AC_ARG_ENABLE([disable-warnings], AS_HELP_STRING([--disable compiler warnings],
 		fi	
 ])
 
+if test "x$CXX" = "xclang++"; then		
+		CPP_STANDARD=17
+    EXTRAFLAGS="-Wno-c++11-narrowing"
+	else
+		EXTRAFLAGS=""	
+fi	
 
 if test "x$CXX" = "xicpc"; then		
 	CXXFLAGS="$CXXFLAGS -O3 -no-prec-div -no-prec-sqrt $WARNINGS_ON -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -std=c++${CPP_STANDARD} -wd1125"
 	CPPFLAGS="$CPPFLAGS -O3 -no-prec-div -no-prec-sqrt $WARNINGS_ON -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE"
 else
-	CXXFLAGS="$CXXFLAGS -DNDEBUG -funroll-loops -O3 -pipe $WARNINGS_ON -fexpensive-optimizations -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -std=c++${CPP_STANDARD}"		
+	CXXFLAGS="$CXXFLAGS -DNDEBUG -funroll-loops -O3 -pipe $WARNINGS_ON -fexpensive-optimizations -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -std=c++${CPP_STANDARD} $EXTRAFLAGS" 		
```

Comment:
Instead of the extra flags, suggest just putting an else if on clang++, add the warning suppression directly (no variable substitution) and leave a comment for why that warning is suppressed.